all-sdk/external-client-configuration.md

```diff
@@ -441,6 +449,9 @@ var client = WorkflowClient.newInstance(stubs, clientOptions);
 * New package and module for `envconfig`.
   * 💭 Why not just in the SDK?
     * We have to take a TOML dependency which is not fair to put on users that don't need this functionality.
+  * ❓ What about "native connection"?
+    * Python and .NET and Ruby will use Rust Core to load this, so TypeScript may have to have two forms too, up to
```

Comment:
TS will only have a lang-side TOML parser. The profile feature has enough value for people to want that in client-only scenarios, so a pure-JS implementation is a must. From that point, it would make no sense to expose both a JS and a Native variant of the "load config" API.